spec/support/authentication_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module AuthenticationHelpers
+  def stub_current_user user
+    ApplicationController.any_instance.stub(:current_user) { user }
+  end
+
+  def stub_authorize_user!
+    ApplicationController.any_instance.stub(:authorize_user!) { true }
```

Comment:
This is really obvious copypasta. You don't use authorize_user! in your application so you don't need to stub it in your tests. Copying pre-existing code is fine, but be careful not to pollute your current project with cruft you don't need, or stuff that makes it look like you are copying blindly.